recipes/tinyxml2/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class Tinyxml2Conan(ConanFile):
+    name = "tinyxml2"
+    license = "Zlib"
+    homepage = "https://github.com/leethomason/tinyxml2"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "TinyXML2 is a simple, small, efficient, C++ XML parser that can be easily integrated into other programs."
+    topics = ("xml", "xml-parsel", "tiny-xml")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_folder = self.name + "-" + self.version
+        os.rename(extracted_folder, self._source_subfolder)
+
+    def _patch_sources(self):
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                              "set(CMAKE_POSITION_INDEPENDENT_CODE ON)", "")
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
```

Comment:
While you're waiting for EAP approval, let's add cmake caching!
```suggestion
        if self._cmake:
            return self._cmake
        self._cmake = CMake(self)
```